module/genericsUtils.ts

```diff
@@ -27,28 +38,25 @@ export async function runMain(apiKey: string, modelKey: string, modelInputs: obj
 }
 
 export async function startMain(apiKey: string, modelKey: string, modelInputs: object = {}): Promise<string>{
-    const callID = await startAPI(apiKey, modelKey, modelInputs)
-    return callID
+    const jsonOut = await startAPI(apiKey, modelKey, modelInputs, true)
+    return jsonOut["callID"]
 }
 
-export async function feedback(apiKey: string, callID: string, feedback: object = {}): Promise<object>{
-    const jsonOut = await feedbackAPI(apiKey, callID, feedback)
-    return jsonOut
-}
 
 export async function checkMain(apiKey: string, callID: string): Promise<object>{
     const jsonOut = await checkAPI(apiKey, callID)
     return jsonOut
 }
 
-const startAPI = async (apiKey: string, modelKey: string, modelInputs: object): Promise<string> => {
-    const urlStart = endpoint.concat("start/v2/")
+const startAPI = async (apiKey: string, modelKey: string, modelInputs: object, startOnly: boolean = false): Promise<any> => {
```

Comment:
to clarify, now runMain() will do check + wait until response but start just makes the call + returns taskID right?